flink-runtime-web/src/test/resources/rest_api_v1.snapshot

```diff
@@ -1,3195 +1,3204 @@
-{
-  "calls" : [ {
-    "url" : "/cluster",
```

Comment:
Seems like we got a line-ending problem here.

Please cherry-pick https://github.com/zentol/flink/commit/207cdf2cadbbbe9e9158164c81d492d924b531e1, and generate the snapshot again (or convert the line-endings manually, whichever is easier for you)

Reply:
Thanks! Done.